metaflow/plugins/argo/argo_workflows.py

```diff
@@ -1102,15 +1113,16 @@ def _container_templates(self):
                 .retry_strategy(
                     times=total_retries,
                     minutes_between_retries=minutes_between_retries,
-                )
-                .metadata(
+                ).metadata(
                     ObjectMeta().annotation("metaflow/step_name", node.name)
                     # Unfortunately, we can't set the task_id since it is generated
                     # inside the pod. However, it can be inferred from the annotation
                     # set by argo-workflows - `workflows.argoproj.io/outputs` - refer
                     # the field 'task-id' in 'parameters'
                     # .annotation("metaflow/task_id", ...)
                     .annotation("metaflow/attempt", retry_count)
+                    # Set labels
+                    .labels(resources.get("labels"))
```

Comment:
Should this be removed?  I think it was getting the labels from the kubernetes resource.  Perhaps this is still useful tho or will be useful in the future?  If not, I am find with removing it.